target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8074-deco-x80-5g.dts

```diff
@@ -0,0 +1,488 @@
+		};
+	};
+};
+
+
+&tlmm {
+	button_pins: button-state {
+		pins = "gpio27";
+		function = "gpio";
+		drive-strength = <2>; /* set 2ma max drive on reset button */
+		bias-pull-up; /* set pullup for reset button */
+	};
+
+	fan_pins: fan-state {
+		pins = "gpio40", "gpio41", "gpio42";
+		function = "gpio";
+		drive-strength = <2>;  /* set 2ma max drive on fan pins */
+		bias-disable; /* disable bias for fan outputs */
+	};
+
+	antenna_pins: antenna-state {
+		ext_antenna_en {
+			pins = "gpio56";
+			function = "gpio";
+			drive-strength = <2>;  /* set 2ma max drive on antenna pins */
+			bias-disable;  /* disable bias for external antenna control */
+		};
+	};
+
+	modem_pins: modem-state {
+		modem_reset {
+			pins = "gpio29";
+			function = "gpio";
+			drive-strength = <2>; /* set 2ma max drive on modem reset */
+			bias-disable; /* disable bias for modem reset control */
+		};
+
+		w_disable {
+			pins = "gpio55";
+			function = "gpio";
+			drive-strength = <2>;  /* set 2ma max drive on modem reset */
+			bias-disable;  /* disable bias for modem wan disable control */
+		};
+	};
+
+	led_pins: led-state {
+		pins = "gpio50", "gpio51", "gpio52";
+		function = "gpio";
+		drive-strength = <8>;  /* set 8ma max drive on led pins */
+		bias-disable; /* disable bias for modem wan disable control */
+	};
+
+	mdio_pins: mdio-pins {
+		mdc {
+			pins = "gpio68";
+			function = "mdc";
+			drive-strength = <8>;
+			bias-pull-up;
+		};
+
+		mdio {
+			pins = "gpio69";
+			function = "mdio";
+			drive-strength = <8>;
+			bias-pull-up;
+		};
+	};
+};
+
+&blsp1_uart5 {
+	status = "okay"; /* enable msm high speed uart */
+};
+
+&cryptobam {
+	status = "okay"; /* enable dma controller */
+};
+
+&crypto {
+	status = "okay";
+};
+
+&prng {
+	status = "okay"; /* enable Pseudo-Random Number Generator */
+};
+
+&qpic_bam {
+	status = "okay";
+};
+
+&qpic_nand {
+	status = "okay"; /* enable nand controller */
+
+	nand@0 {
+		reg = <0>;
+		nand-ecc-strength = <4>;
+		nand-ecc-step-size = <512>;
+		nand-bus-width = <8>;
+
+		partitions {
+			compatible = "qcom,smem-part"; /* define SMEM partition table */
+		};
+	};
+};
+
+&qusb_phy_0 {
+	status = "okay";
+};
+
+&ssphy_0 {
+	status = "okay";
+};
+
+&usb_0 {
+	status = "okay"; /* enable first usb3 controller */
+};
+
+&mdio {
+	status = "okay";
+
+	pinctrl-0 = <&mdio_pins>;
+	pinctrl-names = "default";
+	reset-gpios = <&tlmm 37 GPIO_ACTIVE_LOW>;
+
+	qca8033: ethernet-phy@4 {
+		compatible = "ethernet-phy-ieee802.3-c22"; /* lan phy */
+		reg = <0x4>;
+	};
+
+	qca8081: ethernet-phy@28 {
+		compatible = "ethernet-phy-id004d.d101"; /* wan phy */
+		reg = <28>;
+		reset-deassert-us = <10000>;
+		reset-gpios = <&tlmm 44 GPIO_ACTIVE_LOW>;
+	};
+};
+
+
+&switch {
+	status = "okay";
+
+	switch_lan_bmp = <ESS_PORT6>; /* lan port bitmap */
+	switch_wan_bmp = <ESS_PORT5>; /* wan port bitmap */
+	switch_mac_mode1 = <MAC_MODE_SGMII_CHANNEL0>; /* mac mode for uniphy instance1*/
```

Comment:
If QCA8081 is connected here then use `MAC_MODE_SGMII_PLUS`

Reply:
Ok thanks I have force pushed updates hopefully I have resolved all issues now